curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/62017f47c87be96457c49f46.md

```diff
@@ -39,7 +39,8 @@ Your `span[class~="sr-only"]` selector should have the `-webkit-clip-path` prope
 
 ```js
 // this one gets removed apparently
-assert(code.includes('-webkit-clip-path: inset(50%) !important;'));
+const webkitRegex = /[\s]*-webkit-clip-path[\s]*:[\s]*inset\([\s]*50%[\s]*\) !important;/
```

Comment:
The leading `[\s]*` is redundant and can be removed. If you want the regex to be bulletproof instead, change it to `[\s]+`, so `garbage-webkit-...` wouldn't match.
The last space should also be potentially matching several spaces, probably none works as well.

Reply:
Okay I'm removing the leading the leading `[\s]*`

Reply:
Why not make it airtight instead?